### DISCUSSION TITLE
System Volume Control

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#903] [Ideas]
I like the volume control function of 7+ taskbar tweak.
This function is simply an idea.
- control : Wheel on searchbox 
- alt+page up down when blank state.

![image](https://user-images.githubusercontent.com/6903107/146154778-d9012f1f-65eb-43f8-8b81-97cf10f974ac.png)

https://stackoverflow.com/questions/13139181/how-to-programmatically-set-the-system-volume

https://github.com/xenolightning/AudioSwitcher

**Comment by jjw24** [comment]
this is a good idea.

Sys plugin perhaps

i think the challenge might be how to let sys plugin know alt+page up or scroll action has occurred on the search window

**Reply by onesounds** [reply]
Basic functions are implemented in sys plugin, and scrolling must be built in. 
It should be a structure that calls sys plugin during a scroll operation.
(When using these functions, we need to consider a menu that sets which plug-in to call. for example, user can set scroll to  brightness plugin, in this case, wheel adjust monitor brightness)

It seems that the part that controls the system sound needs something like a separate nuget or something.
Anyway, since it is not an important function now, let's consider implementing it when we can afford it in the future.
